Src/DependencyCollector/NetCore/DependencyCollectorDiagnosticListener.cs

```diff
@@ -27,38 +27,37 @@ public class DependencyCollectorDiagnosticListener : IObserver<DiagnosticListene
         /// <returns>
         /// An IDisposable that can be disposed to disable the DependencyCollectorDiagnosticListener.
         /// </returns>
-        public static IDisposable Enable(TelemetryConfiguration configuration = null)
+        public static IDisposable Enable(TelemetryConfiguration configuration = null, IEnumerable<string> excludedDomains = null)
```

Comment:
@daschult Who's job is it to dispose of the IDisposable that Enable returns?  Would the .NET Core SDK need to keep a reference around until the application is torn down and remember to dispose of it?